source/tutorial.rst

```diff
@@ -57,8 +57,19 @@ Control structures
 Functions
 ^^^^^^^^^^^^
 
-``Work in progress!``
+The way to define functions is similar as it is in other languages.
+
+Note that the return type must be defined if you intend to return anything from the function. If the return type is not specified, the function is assumed to be of type ``void``, so no ``return`` statement is allowed inside the function.
+
```

Comment:
Do you want to add that functions need to be defined at the top of the file? Perhaps we can add it in for now and remove it later if we end up changing our implementation.

Reply:
yeah sure will do that